mongodb/leafygreen-ui
# Adam/tsconfig

Updates TSConfig settings

1. Creates package `@lg-tools/config`
2. Extends `@lg-tools/config/package.tsconfig.json` in every package
3. Adds relative paths to each package's tsconfig
4. Updates `build:ts` job to build each package's TS separately (instead of all at once as a single project)

## How to test
Ensure `yarn install` and `yarn build` work as expected, and output `.js` and `.d.ts` files in `dist`
